include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -5,10 +5,16 @@
 
 #include "core/common/common.h"
 #include "core/graph/graph_viewer.h"
-#include "core/optimizer/rewrite_rule.h"
 
 namespace onnxruntime {
 
+enum TransformerLevel {
+  Default = 1,
+  Optional_L1 = 2,
```

Comment:
>Optional_L1 [](start = 2, length = 11)

Why does the level name need 'optional' in it? Isn't whether it's optional or not a property of the transformer manager rather than the level itself?